<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaVideo</class>
 <widget class="QWidget" name="TelaVideo">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="labelNome">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelQtdeFrames">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="ssButton">
           <property name="text">
            <string>Screenshoot</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="vsButton">
           <property name="text">
            <string>Initial View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Rotate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="moveOption">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Grab</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>-1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="xSlider">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ySlider">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="zSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Zoom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="zoomSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Point Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="pointSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="labelTempo">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="timeSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="bt_help">
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/assets/bt_hp.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_model">
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/assets/bt_md.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_video">
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/assets/bt_vd.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_camera">
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/assets/bt_cm.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>52</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="CanvasVideo" name="canvasVideo" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CanvasVideo</class>
   <extends>QWidget</extends>
   <header>CanvasVideo.h</header>
   <container>1</container>
   <slots>
    <slot>setZoom(int)</slot>
    <slot>setPoint(int)</slot>
    <slot>screenshoot()</slot>
    <slot>visaoInicial()</slot>
    <slot>setFrame(int)</slot>
    <slot>setXmovement(int)</slot>
    <slot>setYmovement(int)</slot>
    <slot>setZmovement(int)</slot>
    <slot>setMoveOption(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>zoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setZoom(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setPoint(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ssButton</sender>
   <signal>clicked()</signal>
   <receiver>canvasVideo</receiver>
   <slot>screenshoot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setFrame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vsButton</sender>
   <signal>clicked()</signal>
   <receiver>canvasVideo</receiver>
   <slot>visaoInicial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setXmovement(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setYmovement(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setZmovement(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveOption</sender>
   <signal>stateChanged(int)</signal>
   <receiver>canvasVideo</receiver>
   <slot>setMoveOption(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>screenshoot()</slot>
 </slots>
</ui>
